lib/moon/design/form.ex

```diff
@@ -14,6 +14,7 @@ defmodule Moon.Design.Form do
   prop(autocomplete, :string, default: "on", values: ["on", "off"])
   prop(target, :any)
   prop(trigger_action, :any)
+  prop(is_horizontal, :boolean, default: false)
```

Comment:
also - maybe it's a good point to add `@doc` attributes for props? otherwise these props should be added on a component's page props table

Reply:
oh! we don't have a form page ((((((